.github/actions/spelling/patterns.txt

```diff
@@ -15,3 +15,6 @@ data:[a-zA-Z=;,/0-9+-]+
 \b[0-9a-f]{40}\b
 \b([A-Za-z])\1{3,}\b
 El proyecto .* diferentes
+# Package family names and product codes in tests
+PackageFamilyName: \S+
+uninstallResultStr\.find\("[^"]+"\)
```

Comment:
I personally think a regex pattern for any pfn would be better and future proof, instead of every place we'll be referencing a pfn

Reply:
Sure. I just don't have a good idea how to define them.

Reply:
Sure. I just don't know offhand how to match them. Hopefully it shouldn't be too hard.